integration-tests/tests/di_diun.rs

```diff
@@ -4,10 +4,12 @@ use std::time::Duration;
 use common::{Binary, LogSide, TestContext};
 
 use anyhow::{bail, Context, Result};
+use serial_test::serial;
```

Comment:
This (or any other way to execute these tests sequentially) is needed because otherwise the `test_diun` test finds two vouchers instead of one and it fails.

Reply:
Ah, this would be because both tests execute a manufacturing server, and I guess the one from `test_device_credentials_already_active` starts a bit later, so the other one takes both requests.
We should instead make sure that `TestContext` syncs between multiple instances of itself, so that new test servers from other `TestContext`'s get different ports.
I'll try to send a PR for this in a few minutes, because serializing tests is not something I'd like to get into: integration tests can get quite long at some point.

Reply:
#105 should fix that, and mean we no longer have to run them serially.

Reply:
Cool, removed.